Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryClientImpl.java

```diff
@@ -2,47 +2,19 @@
 
 import java.util.Random;
 
-public class TelemetryClient
+public class TelemetryClientImpl extends TelementryConnectionImpl implements TelementryClient

```

Comment:
lets not extend TelementryConnectionImpl instead use its object
It violates Liskov Substitution Principle
Ref: https://www.baeldung.com/java-liskov-substitution-principle